### ISSUE TITLE
Sort order for documents in Browse-Documents listing

### ISSUE THREAD

**Issue by wehooper** [IUBLibTech/newton_chymistry#36]
On the current (XTF-P4) site, when the user clicks on "Browse Manuscripts," the queryParser.xsl sheet determines the order of results in template match ="/", at or around line 48.

It uses three components, sort-settlement, sort-collection, and  sort-shelfmark, which all defined in preFilter.xsl. The preFilter definitions draw from the settlement, collection, and shelfmark elements in the document TEI headers.

Basically a few years we decided that the documents would sorted by location, collection, and then shelfmark. The collection and shelfmark are fairly straightforward, but the location can present some problems because there are two Cambridge settlements, one in the UK, the other in Massachusetts, and the UK should come first, UK < USA.
